public/app/plugins/datasource/influxdb/components/editor/query/influxql/InfluxStartPage.tsx

```diff
@@ -2,10 +2,10 @@ import React, { PureComponent } from 'react';
 
 import { QueryEditorHelpProps } from '@grafana/data';
 
-import InfluxCheatSheet from './InfluxCheatSheet';
+import { InfluxCheatSheet } from './InfluxCheatSheet';
 
 export default class InfluxStartPage extends PureComponent<QueryEditorHelpProps> {
   render() {
-    return <InfluxCheatSheet onClickExample={this.props.onClickExample} />;
+    return <InfluxCheatSheet />;
```

Comment:
Why a function component would not be posssible here?

I was thinking that we could just reexport `InfluxCheatSheet` here

Reply:
You are right about that. I was just cleaning the code but this is something we can do too.

Reply:
https://github.com/grafana/grafana/pull/69669/commits/cb82017ea341fec2d801e444584e01050e2f8f92